lib/pdk/validators/base_validator.rb

```diff
@@ -25,17 +25,29 @@ def self.parse_targets(options)
                     options[:targets]
                   end
 
-        targets.map { |target|
+        skipped = []
```

Comment:
It would be great to add yardoc to describe what the method now returns; and also to briefly explain what _skipped_ and _invalid_ mean